tools/apkdiff/apkdiff.csproj

```diff
@@ -53,9 +54,9 @@
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
     <PackageReference Include="Mono.Options" Version="6.6.0.161" />
     <PackageReference Include="Mono.Posix.NETStandard" Version="1.0.0" />
```

Comment:
I think we can remove this `PackageReference`, and the corresponding `Reference` above. See https://github.com/xamarin/xamarin-android/pull/4267 for context.

Reply:
@pjcollins LibZipSharp uses Mono.Posix.NETStandard. So it should get pulled in automatically...